scripts/rollup/build.js

```diff
@@ -584,7 +591,9 @@ function handleRollupWarning(warning) {
     return;
   }
 
-  if (typeof warning.code === 'string') {
+  if (warning.code === 'CIRCULAR_DEPENDENCY') {
+    // Ignored
```

Comment:
Why?

Reply:
i.e. what's the concrete code path that's breaking and can we fix it first in a separate PR?

Reply:
This warning was introduced in newer Rollup version. The solution is to ignore it in `handleRollupWarning`. 

Simplified example here.
https://github.com/rollup/rollup/issues/2271#issuecomment-475540827

These 3 lines of code is very much related to upgrading to a newer Rollup version.

Reply:
@kunukn Show these warnings please. They need to be checked and probably fixed.

Reply:
I believe the circular dependency has always been there for the React source code. Rollup now generates warning when detected. I am uncertain if `circular dependency` is something that needs to be fixed for the React source code. 

I can change the code to this if that is better?
```jsx
  if (warning.code === 'CIRCULAR_DEPENDENCY') {
    console.warn(warning.message || warning);
  }
```

But this gives this noise when building.
Here is an example for `yarn build react-dom/index --type=UMD_PROD`

```
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
```

Reply:
The is the full circular dependency warning when applying `yarn build`

```
 BUILDING  react.development.js (umd_dev)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  react.development.js (umd_dev)

 BUILDING  react.production.min.js (umd_prod)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  react.production.min.js (umd_prod)

 BUILDING  react.profiling.min.js (umd_profiling)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  react.profiling.min.js (umd_profiling)

 BUILDING  react.development.js (node_dev)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  react.development.js (node_dev)

 BUILDING  react.production.min.js (node_prod)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  react.production.min.js (node_prod)

 BUILDING  React-dev.js (fb_www_dev)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  React-dev.js (fb_www_dev)

 BUILDING  React-prod.js (fb_www_prod)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  React-prod.js (fb_www_prod)

 BUILDING  React-profiling.js (fb_www_profiling)
Circular dependency: packages/shared/warning.js -> packages/react/src/ReactSharedInternals.js -> packages/react/src/ReactDebugCurrentFrame.js -> packages/shared/getComponentName.js -> packages/shared/ReactLazyComponent.js -> packages/shared/warning.js
 COMPLETE  React-profiling.js (fb_www_profiling)

 BUILDING  react-dom.development.js (umd_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-dom.development.js (umd_dev)

 BUILDING  react-dom.production.min.js (umd_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-dom.production.min.js (umd_prod)

 BUILDING  react-dom.profiling.min.js (umd_profiling)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-dom.profiling.min.js (umd_profiling)

 BUILDING  react-dom.development.js (node_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-dom.development.js (node_dev)

 BUILDING  react-dom.production.min.js (node_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-dom.production.min.js (node_prod)

 BUILDING  react-dom.profiling.min.js (node_profiling)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-dom.profiling.min.js (node_profiling)

 BUILDING  ReactDOM-dev.js (fb_www_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactDOM-dev.js (fb_www_dev)

 BUILDING  ReactDOM-prod.js (fb_www_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactDOM-prod.js (fb_www_prod)

 BUILDING  ReactDOM-profiling.js (fb_www_profiling)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactDOM-profiling.js (fb_www_profiling)

 BUILDING  react-dom-test-utils.development.js (umd_dev)
 COMPLETE  react-dom-test-utils.development.js (umd_dev)

 BUILDING  react-dom-test-utils.production.min.js (umd_prod)
 COMPLETE  react-dom-test-utils.production.min.js (umd_prod)

 BUILDING  react-dom-test-utils.development.js (node_dev)
 COMPLETE  react-dom-test-utils.development.js (node_dev)

 BUILDING  react-dom-test-utils.production.min.js (node_prod)
 COMPLETE  react-dom-test-utils.production.min.js (node_prod)

 BUILDING  ReactTestUtils-dev.js (fb_www_dev)
 COMPLETE  ReactTestUtils-dev.js (fb_www_dev)

 BUILDING  react-dom-unstable-native-dependencies.development.js (umd_dev)
 COMPLETE  react-dom-unstable-native-dependencies.development.js (umd_dev)

 BUILDING  react-dom-unstable-native-dependencies.production.min.js (umd_prod)
 COMPLETE  react-dom-unstable-native-dependencies.production.min.js (umd_prod)

 BUILDING  react-dom-unstable-native-dependencies.development.js (node_dev)
 COMPLETE  react-dom-unstable-native-dependencies.development.js (node_dev)

 BUILDING  react-dom-unstable-native-dependencies.production.min.js (node_prod)
 COMPLETE  react-dom-unstable-native-dependencies.production.min.js (node_prod)

 BUILDING  ReactDOMUnstableNativeDependencies-dev.js (fb_www_dev)
 COMPLETE  ReactDOMUnstableNativeDependencies-dev.js (fb_www_dev)

 BUILDING  ReactDOMUnstableNativeDependencies-prod.js (fb_www_prod)
 COMPLETE  ReactDOMUnstableNativeDependencies-prod.js (fb_www_prod)

 BUILDING  react-dom-server.browser.development.js (umd_dev)
 COMPLETE  react-dom-server.browser.development.js (umd_dev)

 BUILDING  react-dom-server.browser.production.min.js (umd_prod)
 COMPLETE  react-dom-server.browser.production.min.js (umd_prod)

 BUILDING  react-dom-server.browser.development.js (node_dev)
 COMPLETE  react-dom-server.browser.development.js (node_dev)

 BUILDING  react-dom-server.browser.production.min.js (node_prod)
 COMPLETE  react-dom-server.browser.production.min.js (node_prod)

 BUILDING  ReactDOMServer-dev.js (fb_www_dev)
 COMPLETE  ReactDOMServer-dev.js (fb_www_dev)

 BUILDING  ReactDOMServer-prod.js (fb_www_prod)
 COMPLETE  ReactDOMServer-prod.js (fb_www_prod)

 BUILDING  react-dom-server.node.development.js (node_dev)
 COMPLETE  react-dom-server.node.development.js (node_dev)

 BUILDING  react-dom-server.node.production.min.js (node_prod)
 COMPLETE  react-dom-server.node.production.min.js (node_prod)

 BUILDING  react-dom-unstable-fizz.browser.development.js (umd_dev)
 COMPLETE  react-dom-unstable-fizz.browser.development.js (umd_dev)

 BUILDING  react-dom-unstable-fizz.browser.production.min.js (umd_prod)
 COMPLETE  react-dom-unstable-fizz.browser.production.min.js (umd_prod)

 BUILDING  react-dom-unstable-fizz.browser.development.js (node_dev)
 COMPLETE  react-dom-unstable-fizz.browser.development.js (node_dev)

 BUILDING  react-dom-unstable-fizz.browser.production.min.js (node_prod)
 COMPLETE  react-dom-unstable-fizz.browser.production.min.js (node_prod)

 BUILDING  react-dom-unstable-fizz.node.development.js (node_dev)
 COMPLETE  react-dom-unstable-fizz.node.development.js (node_dev)

 BUILDING  react-dom-unstable-fizz.node.production.min.js (node_prod)
 COMPLETE  react-dom-unstable-fizz.node.production.min.js (node_prod)

 BUILDING  ReactDOMFizzServer-dev.js (fb_www_dev)
 COMPLETE  ReactDOMFizzServer-dev.js (fb_www_dev)

 BUILDING  ReactDOMFizzServer-prod.js (fb_www_prod)
 COMPLETE  ReactDOMFizzServer-prod.js (fb_www_prod)

 BUILDING  react-art.development.js (umd_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-art.development.js (umd_dev)

 BUILDING  react-art.production.min.js (umd_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-art.production.min.js (umd_prod)

 BUILDING  react-art.development.js (node_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-art.development.js (node_dev)

 BUILDING  react-art.production.min.js (node_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-art.production.min.js (node_prod)

 BUILDING  ReactART-dev.js (fb_www_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactART-dev.js (fb_www_dev)

 BUILDING  ReactART-prod.js (fb_www_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactART-prod.js (fb_www_prod)

 BUILDING  ReactNativeRenderer-dev.js (rn_fb_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactNativeRenderer-dev.js (rn_fb_dev)

 BUILDING  ReactNativeRenderer-prod.js (rn_fb_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactNativeRenderer-prod.js (rn_fb_prod)

 BUILDING  ReactNativeRenderer-profiling.js (rn_fb_profiling)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactNativeRenderer-profiling.js (rn_fb_profiling)

 BUILDING  ReactNativeRenderer-dev.js (rn_oss_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactNativeRenderer-dev.js (rn_oss_dev)

 BUILDING  ReactNativeRenderer-prod.js (rn_oss_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactNativeRenderer-prod.js (rn_oss_prod)

 BUILDING  ReactNativeRenderer-profiling.js (rn_oss_profiling)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactNativeRenderer-profiling.js (rn_oss_profiling)

 BUILDING  ReactFabric-dev.js (rn_fb_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactFabric-dev.js (rn_fb_dev)

 BUILDING  ReactFabric-prod.js (rn_fb_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactFabric-prod.js (rn_fb_prod)

 BUILDING  ReactFabric-profiling.js (rn_fb_profiling)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactFabric-profiling.js (rn_fb_profiling)

 BUILDING  ReactFabric-dev.js (rn_oss_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactFabric-dev.js (rn_oss_dev)

 BUILDING  ReactFabric-prod.js (rn_oss_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactFabric-prod.js (rn_oss_prod)

 BUILDING  ReactFabric-profiling.js (rn_oss_profiling)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactFabric-profiling.js (rn_oss_profiling)

 BUILDING  react-test-renderer.development.js (umd_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-test-renderer.development.js (umd_dev)

 BUILDING  react-test-renderer.production.min.js (umd_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-test-renderer.production.min.js (umd_prod)

 BUILDING  react-test-renderer.development.js (node_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-test-renderer.development.js (node_dev)

 BUILDING  react-test-renderer.production.min.js (node_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-test-renderer.production.min.js (node_prod)

 BUILDING  ReactTestRenderer-dev.js (fb_www_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  ReactTestRenderer-dev.js (fb_www_dev)

 BUILDING  react-test-renderer-shallow.development.js (umd_dev)
 COMPLETE  react-test-renderer-shallow.development.js (umd_dev)

 BUILDING  react-test-renderer-shallow.production.min.js (umd_prod)
 COMPLETE  react-test-renderer-shallow.production.min.js (umd_prod)

 BUILDING  react-test-renderer-shallow.development.js (node_dev)
 COMPLETE  react-test-renderer-shallow.development.js (node_dev)

 BUILDING  react-test-renderer-shallow.production.min.js (node_prod)
 COMPLETE  react-test-renderer-shallow.production.min.js (node_prod)

 BUILDING  ReactShallowRenderer-dev.js (fb_www_dev)
 COMPLETE  ReactShallowRenderer-dev.js (fb_www_dev)

 BUILDING  react-noop-renderer.development.js (node_dev)
 COMPLETE  react-noop-renderer.development.js (node_dev)

 BUILDING  react-noop-renderer.production.min.js (node_prod)
 COMPLETE  react-noop-renderer.production.min.js (node_prod)

 BUILDING  react-noop-renderer-persistent.development.js (node_dev)
 COMPLETE  react-noop-renderer-persistent.development.js (node_dev)

 BUILDING  react-noop-renderer-persistent.production.min.js (node_prod)
 COMPLETE  react-noop-renderer-persistent.production.min.js (node_prod)

 BUILDING  react-noop-renderer-server.development.js (node_dev)
 COMPLETE  react-noop-renderer-server.development.js (node_dev)

 BUILDING  react-noop-renderer-server.production.min.js (node_prod)
 COMPLETE  react-noop-renderer-server.production.min.js (node_prod)

 BUILDING  react-reconciler.development.js (node_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-reconciler.development.js (node_dev)

 BUILDING  react-reconciler.production.min.js (node_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-reconciler.production.min.js (node_prod)

 BUILDING  react-reconciler-persistent.development.js (node_dev)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-reconciler-persistent.development.js (node_dev)

 BUILDING  react-reconciler-persistent.production.min.js (node_prod)
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHotReloading.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactUpdateQueue.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberNewContext.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactUpdateQueue.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactChildFiber.js -> packages/react-reconciler/src/ReactFiberClassComponent.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberReconciler.js -> packages/react-reconciler/src/ReactFiberRoot.js -> packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberReconciler.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHooks.js -> packages/react-reconciler/src/ReactFiberBeginWork.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberHydrationContext.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiber.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiber.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberBeginWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberCompleteWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberCommitWork.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberThrow.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js
Circular dependency: packages/react-reconciler/src/ReactFiberDevToolsHook.js -> packages/react-reconciler/src/ReactFiberWorkLoop.js -> packages/react-reconciler/src/ReactFiberDevToolsHook.js
 COMPLETE  react-reconciler-persistent.production.min.js (node_prod)
```

Reply:
Put it in details tag please.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary